test/univariate/continuous/johnsonsu.jl

```diff
@@ -0,0 +1,127 @@
+@testset "JohnsonSU" begin
+
+    d1 = JohnsonSU(0.0, 10.0, -2.0, 3.0)
+    @test d1 isa JohnsonSU{Float64}
+    @test params(d1) == (0.0, 10.0, -2.0, 3.0)
+    @test shape(d1) == 0.0
+    @test scale(d1) == 10.0
+    @test partype(d1) === Float64
+    @test eltype(d1) === Float64
+    @test rand(d1) isa Float64
+
+    @test median(d1) == quantile(d1, 0.5)
+    x = quantile.(d1, [0.25, 0.45, 0.60, 0.80, 0.90])
+    @test all(cdf.(d1, x) .≈ [0.25, 0.45, 0.60, 0.80, 0.90])
+    y = cquantile.(d1, [0.25, 0.45, 0.60, 0.80, 0.90])
+    @test all(ccdf.(d1, y) .≈ [0.25, 0.45, 0.60, 0.80, 0.90])
+
+    @test mean(d1) ≈ 7.581281
+    @test var(d1) ≈ 19.1969485
+    @test skewness(d1) ≈ 0.7780949
+    @test kurtosis(d1) ≈ 0.447512937
+
+    d1 = JohnsonSU(10.0f0, 10.0f0, 1.0f0, 3.0f0)
+    @test d1 isa JohnsonSU{Float32}
+    @test params(d1) == (10.0f0, 10.0f0, 1.0f0, 3.0f0)
+    @test shape(d1) == 10.0f0
+    @test scale(d1) == 10.0f0
+    @test partype(d1) === Float32
+    @test eltype(d1) === Float64
+    @test rand(d1) isa Float64
+
+    d1 = JohnsonSU(1.0, 1, 0, 1)
+    d1 = Base.convert(JohnsonSU{Float64}, JohnsonSU())
```

Comment:
It's not clear what the purpose of this call is. If it's to exercise the `convert` method that was added, it would be better to directly test its behavior. For example,
```julia
d = JohnsonSU(1.0, 1, 0, 1)
@test convert(Johnson{Float64}, d) === d
@test convert(Johnson{Int}, d) isa Johnson{Int}
```

Reply:
I see. I forgot why I left them un`test`ed 😕 Will add your suggestions!